src/kemal/static_file_handler.cr

```diff
@@ -27,7 +27,7 @@ module Kemal
         return
       end
 
-      expanded_path = File.expand_path(request_path, "/")
+      expanded_path = request_path
```

Comment:
Variable name stopped corresponding to its value. Btw, wasn't the expansion needed after all?

Reply:
Taken this from @hertzDevil suggestion

Reply:
What suggestion?

Reply:
https://github.com/kemalcr/kemal/pull/657#issuecomment-1630850329

Reply:
The suggestion is different then the code committed here...?

Reply:
True that. I took the inspiration and made it work 👍